compiler/rustc_codegen_llvm/src/gotoc/cbmc/goto_program/stmt.rs

```diff
@@ -187,6 +187,28 @@ impl Stmt {
         Stmt::assert(Expr::bool_false(), msg, loc)
     }
 
+    /// RMC-specific function to sanity check expected components of in code
```

Comment:
Lets make this generic, to make this easier to split into its own package.

Reply:
Done!